uPortal-webapp/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -52,7 +52,8 @@
         position:absolute;
         overflow-x: hidden;
         overflow-y: scroll;
-        z-index: 10000;
+        /* Needed as dropdown-menu - based on bootstrap variable @zindex-dropdown */
+        z-index: 1000;
```

Comment:
Could this directly use `@zindex-dropdown`?

Reply:
It's into a jsp, so it won't be possible as this var can be applied on css build
My idea was more to apply a dropdown css class (reusing uPortal skin one), but there is too much to check ...

Reply:
Outside of https://github.com/Jasig/uPortal/blob/b3315872cb9c5b5b578934c0b1e612b1e5875de0/uPortal-webapp/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp#L32-L37
It looks like all of this could be moved to less.
But that would be for another PR.

Reply:
Ok I see, yes this could be moved into less files.
But I don't understand the need to have portlet execution id into css tags, is there an interest ? (it's for js only when querying a dom element)